tests/pytorch/test_pytorch_model_export.py

```diff
@@ -738,6 +758,30 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_load_model_succeeds_when_data_is_model_file_instead_of_directory_saved_as_state_dict(
```

Comment:
Can we also rename the function name since it doesn't match what it actually tests?

Reply:
Done. simplified the names.